httpretty/__init__.py

```diff
@@ -55,6 +55,14 @@ def last_request():
     """returns the last request"""
     return httpretty.last_request
 
+def all_requests():
+    """Returns list of all requests made"""
+    return httpretty.latest_requests
+
+def request_count():
+    """returns total number of requests made"""
+    return len(all_requests())
```

Comment:
why bother with this sugaring at all if `len(requests())` is not that much longer and makes things explicit?